features/admin/admin_journey.feature

```diff
@@ -47,12 +49,27 @@ Scenario: As an admin user who has logged in to Digital Marketplace, I wish to s
   And I click the search button for 'supplier_name_prefix'
   Then I am presented with the 'Suppliers' page for all suppliers starting with 'DM Functional Test Supplier'
 
+Scenario: As an admin user who has logged in to Digital Marketplace, I wish to navigate to the Supplier Users page via the supplier(s) by supplier name prefix page
+  Given I am logged in as a 'Administrator' and navigated to the 'Suppliers' page by searching on suppliers by name prefix 'DM Functional Test Supplier'
+  When I click the 'Users' link for the supplier 'DM Functional Test Supplier'
+  Then I am presented with the 'Users' page for the supplier 'DM Functional Test Supplier'
+
+Scenario: As an admin user who has logged in to Digital Marketplace, I wish to navigate to the Supplier Services page via the supplier(s) by supplier name prefix page
+  Given I am logged in as a 'Administrator' and navigated to the 'Suppliers' page by searching on suppliers by name prefix 'DM Functional Test Supplier'
+  When I click the 'Services' link for the supplier 'DM Functional Test Supplier'
+  Then I am presented with the 'Services' page for the supplier 'DM Functional Test Supplier'
+  And I can see all listings ordered by lot name followed by listing name
```

Comment:
Should this be `I can see all services belonging to that supplier...` - just to be clear it is not *all * listings. (I think `services` is a better name than `listings` here, although stick with `listings` if it's used everywhere else.